### PR TITLE
Backport 1213

### PR DESCRIPTION
**Pull request by Ebrithil95**
Backports #1213 to v0.4

### CODE REVIEW

File: core/http/Cargo.toml [hunk 1]
```diff
@@ -22,16 +22,16 @@ private-cookies = ["cookie/secure"]
 smallvec = "0.6"
 percent-encoding = "1"
 hyper = { version = "0.10.13", default-features = false }
-time = "0.1"
+time = "0.2.4"
 indexmap = "1.0"
-rustls = { version = "0.14", optional = true }
+rustls = { version = "0.16", optional = true }
```
Comment (jebrosen) [inline core/http/Cargo.toml]: Changing these versions is a breaking change due to the `ring` dependency. Backporting this to 0.4 would instead require backporting the `reset_delta` to the cookie 0.11 series - @SergioBenitez, is that likely or unlikely to be accepted?

Comment (SergioBenitez) [reply core/http/Cargo.toml]: Happy to accept a backport to cookie 0.11 and subsequently to 0.4.

Comment (Ebrithil95) [reply core/http/Cargo.toml]: I created a PR https://github.com/SergioBenitez/cookie-rs/pull/135

Comment (Ebrithil95) [reply core/http/Cargo.toml]: I have updated cookies to 0.11.2 now

**Comment by Ebrithil95** [comment]
@jebrosen @SergioBenitez Anything else you want me to change?

**Comment by SergioBenitez** [comment]
@Ebrithil95 To backport fixes, you should cherry-pick the relevant commits from the upstream branch, fixing merges along the way as needed to work in the older branch, and finally adding a commit if necessary for non-merge related fixes. We want to preserve as much of the relevant upstream commit as possible, not create brand new commits that are nearly identical.

I've done exactly this and merged it in into the v0.4 branch as 0c56ecc181b66861d2db4fd348e0da17afc60b3e. Thank you so much for initiating this!

**Pull request closed without merging**